vendor/engines/secure_rooms/config/locales/en.yml

```diff
@@ -21,8 +21,8 @@ en:
   order_details:
     notices:
       missing_entry:
-        badge: Missing Entry
-        alert: This order's occupancy does not have an entry time. Please ensure that all times are set and there is a price policy for the date this order was fulfilled.
+        badge: Inferred Account
```

Comment:
ya this is something I'd leave up to user feedback, but as a default for now I think I'll switch to "Missing Entry" + the expanded message